CMakeLists.txt

```diff
@@ -293,6 +293,9 @@ add_library(x86_64-softmmu
     qemu/tcg/tcg.c
     qemu/translate-all.c
 )
+if(NOT UNICORN_BUILD_SHARED)
```

Comment:
can you put a space between `if` and `(`?

Reply:
I can. Just wanna note, that the file right now has quite a few `if(` and `if (`, but stiles are mixed. (but always `endif()` without a space)